user/cowsql/template.py

```diff
@@ -0,0 +1,23 @@
+pkgname = "cowsql"
+pkgver = "1.15.6"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_gen = ["autoreconf", "-i"]
```

Comment:
does this have to be specified? the default is already `autoreconf -fi` so it should be the same unless the forceful overwrite of uptodate breaks it (what error do you get?)

Reply:
same goes below in raft

Reply:
done